spec/board_spec.rb

```diff
@@ -0,0 +1,133 @@
+require './lib/board'
+
+RSpec.describe Board do
+  describe '#initialize' do
+    it 'exists' do
+      board = Board.new
+
+      expect(board).to be_a Board
+    end
+
+    it 'has a column index hash' do
+      #unexpected Rspec hash complaints
+      board = Board.new
+
+      expect(board.col_indices).to be == {
+        "A" => 0, 
+        "B" => 1, 
+        "C" => 2, 
+        "D" => 3, 
+        "E" => 4, 
+        "F" => 5, 
+        "G" => 6
+      }
+    end
+
+    it 'has an array of seven empty column arrays' do
```

Comment:
I know that we are having a display board show up as a string, are the columns for us to keep track of pieces that have already been played?

Reply:
Yes, the arrays are used for both representing the board and keeping track of piece position. The array's act as if they are filled with 'null' pieces which get turned into '.' when we make the string.